lib/core/test/unit/Cardano/Wallet/DB/StateMachine.hs

```diff
@@ -453,7 +453,6 @@ lockstep m@(Model _ ws) c (At resp) = Event
   Generator
 -------------------------------------------------------------------------------}
 
--- NOTE 'concat' reads better.
```

Comment:
Note that this comment wasn't obsolete. It gives a justification for disabling `hlint` below. Hlint suggests to replace use of `concat` with `++`, but `concat` reads better :)

Reply:
cc @jonathanknowles ^^